tests/integration/tier0/verify-latest-kernel-check-with-failed-repoquery/test_verify_kernel_check_passes.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+
+@pytest.mark.latest_kernel_check_with_failed_repoquery
+def test_verify_latest_kernel_check_passes_with_failed_repoquery(shell, convert2rhel):
+    """
+    This test verifies, that failed repoquery is handled correctly.
+    Failed repoquery subsequently caused the latest kernel check to fail.
+    Introduced fixes should get the process past the latest kernel check.
+    """
+
+    repopath = "/etc/yum.repos.d/tainted_repo.repo"
+    tainted_repo_setup = "[tainted-repo]\nname=Tainted repository\nbaseurl=http://download.eng.bos.redhat.com/beakerrepos/client/Fedor36/\nenabled=1\ngpgcheck=0"
+
+    # Add a tainted repository with intentionally bad baseurl so the repoquery fails successfully.
+    with open(repopath, "w") as tainted_repo:
+        tainted_repo.write(tainted_repo_setup)
+
+    assert shell(f"cat {repopath}").output == tainted_repo_setup
```

Comment:
I don't think this is necessary

Reply:
Indeed, that is probably some debuging remnant. Thank you!